databases/barman/Portfile

```diff
@@ -21,13 +21,11 @@ long_description    Barman (Backup and Recovery Manager) is an open-source \
 
 homepage            https://www.pgbarman.org
 
-master_sites        pypi:[string index ${name} 0]/${name}
+checksums           rmd160  19d2c539d9e6543e1c56c3cabac5d55481372657 \
+                    sha256  b9def4ca092426559c7983d10d26d17a1369d44b0057e19df569a9ad1481b470 \
+                    size    1260434
 
-checksums           rmd160  cf2fe82664338a8c3d449ccdf56c77113b1bb08f \
-                    sha256  46e0af093eaf53065b3cfcf3ed5548b1446629d9177ff4941d65c6d96a7ae5e5 \
-                    size    1247335
-
-python.default_version 38
+python.default_version 39
```

Comment:
Please leave python 38 here. Macports still uses python 3.8 as the default until most of the python modules add python 3.9 sub-ports.

Reply:
Updated.